### PR TITLE
HW 4

### PR DESCRIPTION
**Pull request by saharj**

### CODE REVIEW

File: sahar.saharj/hw_5/2_event_delegation_lab/main.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  var $button = $('#new-thing-button');
+  var $newThingInput = $('#new-thing');
+
+  $things.each(function(index, li) {
+    addCrossOffLink($(li));
+    addDeleteLink($(li));
+  });
+
+  $things.hover(addGreyColor, removeGreyColor);
+
```
Comment (Borgaard) [inline sahar.saharj/hw_5/2_event_delegation_lab/main.js:66]: How do you feel about the `addGreyColor()` functionality with this solution?

File: sahar.saharj/hw_6/jquery_ajax_exercise/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+var weatherUrl = "api.openweathermap.org/data/2.5/weather?id=5391997&APPID=";
+var apiKey = "4ec0b3cb7c9603901ba1957a71ad488d";
+
+var url = 'http://' + weatherUrl + apiKey;
+
+$.get(url, function(r) {
+	var temp = ((r.main.temp * 9) / 5) - 459.67;
+	console.log(temp);
+});
```
Comment (Borgaard) [inline sahar.saharj/hw_6/jquery_ajax_exercise/main.js:22]: 👍

File: sahar.saharj/hw_5/2_event_delegation_lab/main.js (hunk 1 above)
Comment (saharj) [reply sahar.saharj/hw_5/2_event_delegation_lab/main.js:66]: It was interesting that `hover` does not work with `on()`, it took me long to finally figure this out. Also I liked that this one function does everything for `mouseIn` and `mouseOut` events.

Comment (Borgaard) [reply sahar.saharj/hw_5/2_event_delegation_lab/main.js:66]: You've found a smart solution - very nice! :)

**Pull request merged**